README.md

```diff
@@ -1378,9 +1378,14 @@ I am sure it will be an awesome journey!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
 
+
+# Hi guys!
+nice to join such a wonderfull jorney! ![Cat](https://gph.is/g/4o6B0Km)
```

Comment:
Need more changes...
According to [markdown images syntax](https://www.markdownguide.org/basic-syntax/#images) if you want to add not just link, but image preview you should use **path to file source**.

<details>
  <summary><b>Warning:</b> click only if you can't handle by your self!</summary>
<code>![funny cat](https://i.giphy.com/media/VbnUQpnihPSIgIXuZv/giphy.webp)</code>
</details>

Reply:
Looks good! Try to be more attentive 👀